runtime/src/bank.rs

```diff
@@ -1899,7 +1899,7 @@ impl Bank {
             .set_in_flight(parent.slot());
         parent.force_flush_accounts_cache();
         let accounts_hash =
-            parent.update_accounts_hash(CalcAccountsHashDataSource::Storages, false, true);
+            parent.update_accounts_hash(CalcAccountsHashDataSource::IndexForTests, false, true);
```

Comment:
Yes. Because since we're using the write cache, we are not writing to append vecs yet. Yet, we need to update the accounts hash. Then, we store more accounts to the same slot during warping. So, we have to use the index version of the hash calculation.